src/adapter/src/coord.rs

```diff
@@ -51,7 +51,7 @@
 //!   objects whose `since` times are >= its `upper`.
 //!
 //! This illustration attempts to show, with time moving left to right, the
-//! relationship between `since` and `upper`.
+//! relationship between `since` and `upper`..
```

Comment:
I think maybe you made this change by mistake?

Reply:
Sorry, this was an extra commit just to get an actual new build to verify this change in CI. Let me remove that again.